src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -82,6 +82,7 @@ public class SystemConfig {
     private String buildNumber = null;
     
     private static final String JVM_TIMER_SERVER_OPTION = "dataverse.timerServer";
+    private static final String JVM_GLOBUS_TASK_MONITORING_OPTION = "dataverse.globus.taskMonitoringServer";
```

Comment:
Can this just be a  JvmSettings entry like GLOBUS_CACHE_MAXAGE? (maybe both should be in a Globus namespace - the cache age is in dataverse.files right now).

Reply:
Yeah, the first part is a no-brainer - moved to JvmSettings. As for creating a new globus namespace for both of these settings, I didn't feel as comfortably messing with an existing setting, so I have them both under dataverse.files, if that's ok.